packages/shared/test/integration/dkg-client.test.ts

```diff
@@ -1,50 +1,51 @@
 import { SecretKey } from '@nucypher/nucypher-core';
-
-import { DkgCoordinatorAgent } from '../../src/agents/coordinator';
 import {
-  fakeCoordinatorRitual,
-  fakeDkgParticipants,
   fakeProvider,
-  fakeRitualId,
+  mockCoordinatorRitual,
+  mockDkgParticipants,
   mockGetParticipants,
-} from '../utils';
+  mockRitualId,
+} from '@nucypher/test-utils';
```

Comment:
I moved test utilities and variables to `@nucypher/test-utils` because we're going to use them in different packages (`pre`, `taco`, `shared`), and having a separate package makes things slightly easier in monorepo than using relative paths.